packages/ckeditor5-upload/src/ui/filedialogbuttonview.ts

```diff
@@ -38,10 +41,17 @@ import View from '@ckeditor/ckeditor5-ui/src/view';
  * @extends module:ui/view~View
  */
 export default class FileDialogButtonView extends View {
+	public buttonView: ButtonView;
+
+	private _fileInputView: FileInputView;
+
+	declare public acceptedType: string;
+	declare public allowMultipleFiles: boolean;
+
 	/**
 	 * @inheritDoc
 	 */
-	constructor( locale ) {
+	constructor( locale: Locale ) {
```

Comment:
```suggestion
    constructor( locale?: Locale ) {
```

It is optional in `super()` and isn't required in this class.